RELEASE.md

```diff
@@ -0,0 +1,253 @@
+
+OVN-Kubernetes is supported until the next major or minor release.
+
+For example, if the current release is `v1.1` it is supported until either `v2.0` is released or `v1.2` is released.
+
+Patch releases will only be created for critical issues that are discovered after the release. In the case a patch release has been created, that release will be supported until the next major or minor release.
+
+For example, if an issue was found in `v1.1.0` it would be fixed in `v1.1.1` and `v1.1.1` would be supported until `v1.2.0` is released. `v1.1.0` would not be supported after `v1.1.1` is released.
+
+## How do we decide on the release number?
+
+The release number is decided by the community in the [community meeting] during the release planning phase.
+
+The release number is a semantic version number.
+
+The release number is incremented by the following rules:
+
+- Major version is incremented for any backwards incompatible changes.
+- Minor version is incremented for new features, but not backwards incompatible changes.
+- Patch version is incremented for bug fixes.
+
+## What is the release process?
+
+The release process is as follows:
+
+### Step 1: Planning
+
+- The release coordinator is elected following the process outlined in [#who-is-responsible-for-the-release].
+- The release date and code freeze date are decided during the [community meeting] following the process outlined in [#how-do-we-decide-on-the-release-date].
+- The payload of the release is decided during the [community meeting], following the process outlined in [#what-goes-into-a-release].
+- The release coordinator will create a tracking issue and GitHub milestone for the release
+- All issues and pull requests that are targeted for the release are added to the GitHub milestone
+- The tracking issue is created using the [release-planning.md](.github/ISSUE_TEMPLATE/release-planning.md) template
+- The tracking issue is assigned to the release coordinator
+- The tracking issue is the source of truth for the current state of the release
+
+### Step 2: Development
+
+- The development phase is the first 2.5 months of the release cycle
+- During this phase the `master` branch is open for new changes
+- There are no restrictions on the type of changes that can be made to the codebase
+  during this phase
+- The normal review process is followed for all changes
+- The release coordinator updates the community on the progress of the release during the [community meeting]
+
+### Step 3: Code Freeze
+
+- The code freeze date is the 2 weeks before the release date
+- During this phase the `master` branch is closed for new changes
+- Any bug fixes or documentation changes must follow the normal review process, but will also require the explicit approval of the release coordinator.
+- OKEPs may be reviewed and merged into the `master` branch during this phase without explicit approval from the release coordinator since they do not affect the release itself.
```

Comment:
does it make sense to clarify that these OKEP's **implementation** must be targeting the **next** release ?

Reply:
no i think that's implied. generally okep authors want their okeps implemented asap ;)
asap however is not guaranteed to be the next release anymore.